doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -138,6 +138,13 @@ LTE link control library
      * Remove the use of the ``CONFIG_LTE_NETWORK_USE_FALLBACK`` Kconfig option.
        Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
        In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
+     * If your application uses the :c:func:`lte_lc_conn_eval_params_get` function, you must use the new :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option.
+     * If your application handles the :c:enumerator:`LTE_LC_EVT_EDRX_UPDATE` event or uses the :c:func:`lte_lc_ptw_set`, :c:func:`lte_lc_edrx_param_set`, :c:func:`lte_lc_edrx_req`, or :c:func:`lte_lc_edrx_get` function, you must use the new :kconfig:option:`CONFIG_LTE_LC_EDRX` Kconfig option.
```

Comment:
For better readability, you might consider the following in cases where you list several functions:

* Use the new :kconfig:option:`CONFIG_LTE_LC_EDRX` Kconfig option if your application handles the ... or uses one of the following functions:

  * function 1
  * function 2

Reply:
Updated.

Reply:
A lot easier to read now, isn't it?

Reply:
Yes!